content/en/examples/controllers/job.yaml

```diff
@@ -8,7 +8,7 @@ spec:
       containers:
       - name: pi
         image: perl
-        command: ["perl",  "-Mbignum=bpi", "-wle", "print bpi(2000)"]
+        command: ["perl", "-Mbignum=bpi", "-wle", "print bpi(1000)"]
```

Comment:
I did find one issue. If you look at https://deploy-preview-34070--kubernetes-io-main-staging.netlify.app/docs/concepts/workloads/controllers/job/ which references this example it says pi is calculated to 2000 digits.  Can you update this page to say 1000 as well?

Reply:
Good point, fixed via f9445734525c2445e1aa16112a0e534b1f7b47d7.